src/test/java/org/opensearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -478,7 +549,6 @@ public void testPasswordRules() throws Exception {
         HttpResponse response = rh
                 .executeGetRequest("_plugins/_security/api/" + CType.INTERNALUSERS.toLCString());
         Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
-        System.out.println(response.getBody());
```

Comment:
Hi Ryan, I was removing that since I believe we generally avoid System.out.println when possible. I don't know if that line snuck through on a previous PR but I don't think it is a pattern we endorse so figured I would remove it.